precompiles/pointer/pointer.go

```diff
@@ -32,6 +34,9 @@ const PointerAddress = "0x000000000000000000000000000000000000100b"
 
 var _ vm.PrecompiledContract = &Precompile{}
 var _ vm.DynamicGasPrecompiledContract = &Precompile{}
+var TopicNativePointerDeployed = crypto.Keccak256Hash([]byte("native_deployed"))
+var TopicCW20PointerDeployed = crypto.Keccak256Hash([]byte("cw20_deployed"))
```

Comment:
if the cw side address is the first topic then I think we may want something like `CW20Deployed(string)` as the event signature that gets hashed as the first topic.

Reply:
oh actually it might be something like `CW20Deployed(string,address)`